src/main/java/org/springframework/data/neo4j/core/mapping/callback/AuditingBeforeBindCallback.java

```diff
@@ -30,7 +32,7 @@
  * @soundtrack Iron Maiden - Iron Maiden
  * @since 6.0
  */
-@API(status = API.Status.INTERNAL, since = "6.0")
+@API(status = STABLE, since = "6.0")
 public final class AuditingBeforeBindCallback implements BeforeBindCallback<Object>, Ordered {
```

Comment:
Do you want to keep this class `public`?

Reply:
Yes. So that people can have a choice either using them directly, via the annotation (`@EnableAuditing…`) or the registrar…). They are different than the id and optimistic locking callbacks as they are not required for a correct function of the whole system.